controllers/funding/10k.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const contentApi = require('../../services/content-api');
+
+function init10k({ router, routeConfig, caseStudySlugs }) {
+    router.get(routeConfig.path, (req, res) => {
+        const lang = req.i18n.__(routeConfig.lang);
+        const currentLocale = req.i18n.getLocale();
+
+        const servePage = caseStudies => {
+            res.render(routeConfig.template, {
+                title: lang.title,
+                description: lang.description || false,
+                copy: lang,
+                caseStudies: caseStudies || []
+            });
+        };
+
+        contentApi
+            .getCaseStudies({
+                locale: currentLocale,
+                slugs: caseStudySlugs
+            })
+            .then(
+                caseStudies => {
+                    servePage(caseStudies);
+                },
+                function() {
```

Comment:
What's this, the error handler? Bit confusing to read (or might be because I need a coffee)?

Reply:
☕️second argument to `.then` is the error handler ☕️

Reply:
I'm going back to bed.